x-pack/legacy/plugins/apm/public/components/app/GlobalHelpExtension/index.tsx

```diff
@@ -4,46 +4,27 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { EuiLink } from '@elastic/eui';
-import { i18n } from '@kbn/i18n';
-import React, { Fragment } from 'react';
-import styled from 'styled-components';
+import React from 'react';
 import url from 'url';
-import { px, units } from '../../../style/variables';
+import { i18n } from '@kbn/i18n';
+import { EuiLink, EuiText } from '@elastic/eui';
 import { useKibanaCore } from '../../../../../observability/public';
 
-const Container = styled.div`
-  margin: ${px(units.minus)} 0;
-`;
-
 export const GlobalHelpExtension: React.SFC = () => {
   const core = useKibanaCore();
 
   return (
-    <Fragment>
-      <Container>
-        <EuiLink
-          href="https://discuss.elastic.co/c/apm"
-          target="_blank"
-          rel="noopener"
-        >
-          {i18n.translate('xpack.apm.feedbackMenu.provideFeedbackTitle', {
-            defaultMessage: 'Provide feedback for APM'
-          })}
-        </EuiLink>
-      </Container>
-      <Container>
-        <EuiLink
-          href={url.format({
-            pathname: core.http.basePath.prepend('/app/kibana'),
-            hash: '/management/elasticsearch/upgrade_assistant'
-          })}
-        >
-          {i18n.translate('xpack.apm.helpMenu.upgradeAssistantLink', {
-            defaultMessage: 'Upgrade assistant'
-          })}
-        </EuiLink>
-      </Container>
-    </Fragment>
+    <EuiText size="s">
+      <EuiLink
+        href={url.format({
+          pathname: core.http.basePath.prepend('/app/kibana'),
+          hash: '/management/elasticsearch/upgrade_assistant'
+        })}
+      >
+        {i18n.translate('xpack.apm.helpMenu.upgradeAssistantLink', {
+          defaultMessage: 'Upgrade assistant'
+        })}
+      </EuiLink>
```

Comment:
Instead of rendering this via `content` I think it should be possible to declare custom links if an applicable type doesn't exist:

```js
{
  linkType: 'custom',
  icon: 'any valid eui icon',
  href: '/app/kibana/management/elasticsearch/upgrade_assistant',
  label: {i18n.translate('xpack.apm.helpMenu.upgradeAssistantLink', {
    defaultMessage: 'Upgrade assistant'
  })}
}
```

To me it seems like `content` should be a very last resort (it's exactly what'll cause inconsistencies again down the line), so giving consumers an option to avoid it would be nice.

Reply:
That being said, I think `upgrade_assistant` should be a dedicated link type.

Reply:
I can certainly return an object instead of React content for this particular link. However, I'm going to leave the decision to make the upgrade assistant link a known link for later discussion. I only see this one usage and there may be a better way to use the Kibana Management page for this type of messaging.

Reply:
Updated